components/pd_client/src/util.rs

```diff
@@ -89,17 +89,17 @@ impl LeaderClient {
     pub fn new(
         env: Arc<Environment>,
         security_mgr: Arc<SecurityManager>,
-        client: PdClient,
+        client_stub: PdClientStub,
```

Comment:
maybe still call it `client`? just rename the type

Reply:
I think use `client_stub` is better than `client` since it makes clear that this is a grpc client stub.